python/lsst/display/firefly/firefly.py

```diff
@@ -479,3 +482,77 @@ def _pan(self, colc, rowc):
         self._lastPan = [colc+0.5, rowc+0.5] # saved for future use in _mtv
         # Firefly's internal convention is first pixel is (0.5, 0.5)
         _fireflyClient.set_pan(plot_id=str(self.display.frame), x=colc, y=rowc)
+
+    # Extensions to the API that are specific to using the Firefly backend
+
+    def getClient(self):
+        """Get the instance of FireflyClient for this display
+
+        Returns:
+        --------
+        `firefly_client.FireflyClient`
+            Instance of FireflyClient used by this display
+        """
+        return self._client
+
+    def clearViewer(self):
+        """Reinitialize the viewer
+        """
+        self._client.reinit_viewer()
+
+    def resetLayout(self):
+        """Reset the layout of the Firefly Slate browser
+
+        Clears the display and adds Slate cells to display image in upper left,
+        plot area in upper right, and plots stretch across the bottom
+        """
+        self.clearViewer()
+        self._client.add_cell(row=2, col=0, width=4, height=2, element_type='tables',
+            cell_id='tables')
+        self._client.add_cell(row=0, col=0, width=2, height=3, element_type='images',
+            cell_id='image-%s' % str(self.display.frame))
+        self._client.add_cell(row=0, col=2, width=2, height=3, element_type='xyPlots',
+            cell_id='plots')
+
+    def overlayFootprints(self, catalog, color='rgba(74,144,226,0.60)',
+                          highlight_color='cyan', select_color='orange',
+                          style='fill', layer_string='detection footprints ',
+                          title='catalog footprints '):
+        """Overlay outlines of footprints from a catalog
+
+        Overlay outlines of LSST footprints from the input catalog. The colors
+        and style can be specified as parameters, and the base color and style
+        can be changed in the Firefly browser user interface.
+
+        Parameters:
+        -----------
+        catalog : `lsst.afw.table.SourceCatalog`
+            Source catalog from which to display footprints.
+        color : `str`
+            Color for footprints overlay
```

Comment:
Should document (or point to documentation) for how colors are encoded into strings.

Reply:
Added color specification to the docstring and to the example notebook.